ooniprobe/Utility/TestUtility.mm

```diff
@@ -144,6 +144,14 @@ + (void)downloadUrlsCallback:(NSData *)data
     successcb(urls);
 }
 
+//TODO where and when to call it?
++ (void)deleteUploadedJsons{
+    for (Measurement *measurement in [Measurement measurementsWithJson]){
+        [measurement getExplorerUrl:^(NSString *measurement_url){
+            [TestUtility removeFile:[measurement getReportFile]];
+        } onError:nil];
```

Comment:
Can you write a test where you fill in, say, two measurements that are in explorer and one that is not, and verify that the files that should be removed are removed and the other is still there? You can operate like this:

1. save the measurement JSON from explorer or the API (I can help with that)

2. create the files and populate the `Measurement`

3. call `deleteUploadedJsons` and then verify that what happened is what we expected